lib/launcher.js

```diff
@@ -424,6 +424,9 @@ class Launcher {
                     timeout: { request: this.healthCheckInterval - 500 },
                     retry: { limit: 0 }
                 }
+                if (this.settings.settings.disableEditor) {
+                    opts.throwHttpErrors = false
+                }
```

Comment:
Should we not simply be disabling health poll or skip out of it if editor is disabled Ben?

I mean this is gonna be polling a non-existent endpoint and getting a 404 (I assume) so should we not just skip out of the health check?

Reply:
No, this is still valid as the request will timeout if the NR instance is overworked before returning a 404

Reply:
I'm nearly tempted to not if block this and just set it on the options for all health check requests

Reply:
Lets go with what you tested for now Ben.